plugins/confirm.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: confirm
+spec:
+  version: v0.1.0
+  homepage: https://github.com/brianpursley/kubectl-confirm
+  shortDescription: Show config, dry-run, and diff, and prompt for confirmation before running a command
```

Comment:
This might not fit "kubectl krew search". Please keep <=50 chars.

Reply:
Ah yeah, I forgot about the limit there, thanks for reminding me.

I changed it to `Dry-run / diff / confirm before running a command`